docs/pages/abide.md

```diff
@@ -333,7 +333,7 @@ number : /^[-+]?\d*(?:[\.\,]\d+)?$/,
 card : /^(?:4[0-9]{12}(?:[0-9]{3})?|5[1-5][0-9]{14}|6(?:011|5[0-9][0-9])[0-9]{12}|3[47][0-9]{13}|3(?:0[0-5]|[68][0-9])[0-9]{11}|(?:2131|1800|35\d{3})\d{11})$/,
 cvv : /^([0-9]){3,4}$/,
 
-// http://www.whatwg.org/specs/web-apps/current-work/multipage/states-of-the-type-attribute.html#valid-e-mail-address
+// //www.whatwg.org/specs/web-apps/current-work/multipage/states-of-the-type-attribute.html#valid-e-mail-address
```

Comment:
Removing the protocol makes it unable to ctrl-click this directly in an IDE. I would propose to revert the removal of the protocil.

Reply:
I still see the versin without the protocol. Did you already revert this change?

Reply:
I was just reviewing the changes here and noticed this one. I'll fix it now